components/style/themes/default.less

```diff
@@ -290,6 +290,10 @@
 @table-padding-vertical: 16px;
 @table-padding-horizontal: 8px;
 
+// Tag
+// --
+@tag-default-bg: @background-color-base;
```

Comment:
The `Tag` has a shader border for visual accessibility which select tag don't have it and `#f7f7f7` looks too light.